minimal-notebook/Dockerfile

```diff
@@ -95,5 +95,16 @@ COPY start-notebook.sh /usr/local/bin/
 COPY jupyter_notebook_config.py /home/$NB_USER/.jupyter/
 RUN chown -R $NB_USER:users /home/$NB_USER/.jupyter
 
+# Fetch additional scripts that run Notebook server in single-user mode (for
+# use with JupyterHub)
+RUN wget \
+    -q https://raw.githubusercontent.com/jupyter/jupyterhub/master/scripts/jupyterhub-singleuser \
```

Comment:
This is the only piece that makes me less confident about this PR being the way to go. JupyterHub isn't sensitive to the versions of anything in these images other than the version of JupyterHub whence came this singleuser script, which really should match the parent JupyterHub.

Pinning to stable (0.5.0) makes sense, though.